domain/account/usecases/get-account.go

```diff
@@ -0,0 +1,23 @@
+package usecases
+
+import (
+	"errors"
+
+	"github.com/Erikaa81/banco-erika/domain/entities"
+)
+
+var (
+	ErrEmptyId = errors.New("the id was not filled")
+)
+
+func (a Account) GetAccount(id string) (entities.Account, error) {
```

Comment:
```suggestion
func (a Account) Get(id string) (entities.Account, error) {
```
como esse método já está no contexto de Account, é redundante dizer que cê tá pegando uma conta. Acredito que só Get baste, assim como cê fez com o List